metadata-ingestion/src/datahub/cli/ingest_cli.py

```diff
@@ -159,8 +159,6 @@ async def run_func_check_upgrade(pipeline: Pipeline) -> None:
                     f"timed out with {e} waiting for version stats to be computed... skipping ahead."
                 )
 
-        sys.exit(ret)
```

Comment:
removed as this makes no sense to exit the cli in case there is some issue here in getting version stats. This is a helper method. If are not logging the exception as an error then does not make sense to exit with the wrong status code either.

Reply:
This has nothing to do with getting version stats.

Reply:
Then why is it in the version stats method?

Reply:
This caused the ingestion to fail in case we failed to get version stats. It makes no sense for ingestion to fail if version stats were not calculated.